Umweltzone/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rename the `values-cs-rCZ` folder to `values-cs` to address all Czech speaking regions.